src/modules/topics/components/topics-view.jsx

```diff
@@ -149,16 +151,19 @@ export default class TopicsView extends Component {
 
   updateFilteredItems(filteredTopics) {
     this.setState({
+      loaded: true,
```

Comment:
The class method `updateFilteredItems` actually gets called almost immediately after mount from within the `FilterSort` component, as a result this spinner most likely would never display.  

A more ideal solution would be to set the `loaded` state in reaction to the actual topics request to augur.